refactoring/initial_person_class.py

```diff
@@ -16,7 +16,8 @@ def add_connection(self, person, relation):
 
     def forget(self, person):
         """Removes any connections to a person"""
-        pass
+        if person in self.connections:
```

Comment:
I like that you've made sure that they exist, in our example we've thrown a specific exception here but this is also another sensible way to deal with it